compiler/rustc_session/src/config/sigpipe.rs

```diff
@@ -16,7 +16,10 @@ pub const SIG_IGN: u8 = 2;
 #[allow(dead_code)]
 pub const SIG_DFL: u8 = 3;
 
-/// `SIG_IGN` has been the Rust default since 2014. See
+/// `DEFAULT` is a separate value here, but is resolved to `SIG_IGN` in
+/// `library/std/src/sys/unix/mod.rs`.
+///
+/// Note that `SIG_IGN` has been the Rust default since 2014. See
 /// <https://github.com/rust-lang/rust/issues/62569>.
 #[allow(dead_code)]
-pub const DEFAULT: u8 = SIG_IGN;
+pub const DEFAULT: u8 = 0;
```

Comment:
I agree `0` is a good value for `DEFAULT`, but can we perhaps move it to the top of the file so that the constants becomes sequential? Or maybe change the value to `4`, but that seems less attractive to me.

We should also update

https://github.com/rust-lang/rust/blob/bb5a01617589b5e3ece5a36435fc285bfd13c9a4/library/std/src/rt.rs#L92

to not say "3 values", but instead say "4 values". Or perhaps even better: rephrase it to not specify an exact count.

Reply:
Done, thanks.